### ISSUE TITLE
Could syncing time be reduced?

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#948]
Syncing takes very long from scratch, and people seem to regularly end up with broken blockchain needing to sync again. Are there any ways of increasing the speed? 

- Multi-threading
- Larger chunks at the same time
- Not running x16r on them
- less signature checks on the first x blocks
- etc.

**Comment by fdoving** [comment]
Related to #807  ?

**Comment by HyperPeek** [comment]
Yes, I think #807 contains the discussion that may be the way to go. A fork for this is no option, I think, and we want the core wallet to contain all blocks. 
From my experience sync time is significantly different on a (entry-level) VPS, local wallet on PC in Linux and local PC on Windows, with VPS being faster than Linux at home and multiple times faster than Windows wallets.
This looks like we are looking into multiple issues here, actually; networking (where block-size like #807 may help) but also CPU restrictions, at least on Windows. In my tests he wallet only ever used one CPU, not sure where that restriction comes from.

**Comment by fdoving** [comment]
We could consider an increase of the default dbcache. It might help a little bit with sync speed.
It is a simple change. The current 450MB was set in bitcoin in 2017, they still have 450.

https://github.com/RavenProject/Ravencoin/blob/9def8ed6b5aa6f8d6aa3a53436197233941d7e17/src/txdb.h#L29
